w3-aggregation.md

```diff
@@ -28,67 +28,116 @@ There are several roles in the authorization flow:
 
 ```mermaid
 sequenceDiagram
-    participant Storefront as 🌐<br/><br/>did:web:web3.storage
-    participant Authority as 🌐<br/><br/>did:web:spade.storage
-
-    Storefront->>Authority: invoke `aggregate/offer`
-    Note left of Authority: Request offer to be queued
-    Authority-->>Storefront: receipt issued
+    actor Agent as <br/><br/>did:key:a...
+    actor Storefront as <br/><br/>did:web:web3.storage
+
+    Agent->>Storefront: invoke `filecoin/add`<br>with:`did:key:aSpace`
+    Note left of Storefront: Request piece to be added to filecoin deal
+    Storefront->>Storefront: invoke `filecoin/add`<br>with:`did:key:filecoinQueue`
+    activate Storefront
+    Storefront-->>Agent: receipt issued as `queued`
+    Storefront->>Storefront: invoke `filecoin/add`<br>with:`did:web:web3.storage`
+    deactivate Storefront
+    Storefront-->>Agent: receipt issued<br>fx: `piece/add`
 ```
 
-### Broker queues the aggregate
+### Storefront offers a piece to aggregate
 
-Once a broker successfully receives the offer of an aggregate, the aggregate gets queued for review. A receipt is created to proof the transition of the offered aggregate state from `null` into `queued`. It is worth mentioning that if an offer is for an aggregate that is already `queued` or `complete` it is ignored.
+Once a Storefront receives a valid piece, it MAY be offered for aggregation, so that it makes its way into a Storage Provider. Aggregation MAY be handled asynchronously, therefore the Aggregator MUST acknowledge a request by issuing a signed receipt.
 
-This receipt MUST have link to a followup task (using `.fx.join` field) that either succeeds (if the aggregate was added into a deal) or fails (if the aggregate was determined to be invalid) so that it's receipt COULD be looked up using it.
+Once an Aggregator successfully receives a piece offer, the piece gets queued for aggregation. A receipt is created to proof the transition of the offered aggregate state from `null` into `queued`. It is worth mentioning that if an offer is for a piece that is already `queued` or `accepted`, it has no effect.
 
-> Note: Aggregator MAY have several intermediate steps and states it transitions through, however those intentionally are not captured by this protocol, because storefront will take no action until success / failure condition is met.
+This receipt MUST have link to a followup task (using `.fx.join` field) that either succeeds (if the piece was added into an aggregate) or fails, so that its receipt COULD be looked up using it.
 
-### Broker reviews and handles the aggregate
+After an aggregator dequeues the piece, it will be included into an aggregate to be offered for Storage Providers. A receipt is created to proof the transition of the aggregate state from `queued` into `accepted` or `rejected`. If successful, a inclusion proof should also be provided.
```

Comment:
```suggestion
After an aggregator dequeues the piece, it will be included into an aggregate to be offered to Storage Providers. A receipt is created to prove the transition of the aggregate state from `queued` to `accepted` or `rejected`. If successful, an inclusion proof SHOULD also be provided.
```

Inclusion proof should be MUST?